lib/scrabble.rb

```diff
@@ -10,4 +13,43 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(letter)
+    if letter == nil
+      0
+    elsif letter.length == 1
+      point_values[letter.upcase]
+    else
+      new_array = letter.upcase.split('')
+      conversion = new_array.map do |i|
+        point_values[i]
+        end
+      conversion.sum
+    end
+
+  end
+
+  def score_with_multipliers(letter, letter_score, word_score = 1)
```

Comment:
The logic here looks really good, but the method is doing quite a bit - try to keep your methods to a single responsibility!